em_workflows/utils/flow.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+Utilities workflows:
+-------------------
+These workflows are created simply to aid developers.
+
+- Use #1: Run the tests regularly.
+  Test input files can be of big size which can be hard to manage. Since most of the tests are normally executed on
+  HPC over local machine, it is deemed okay have a prefect scheduler to run the tests for us.
+"""
+from pathlib import Path
+import subprocess
+from time import time
+
+from prefect import flow, task
+from prefect.artifacts import create_markdown_artifact
+
+from em_workflows.config import Config
+from em_workflows.utils import utils
+
+RECENT_FILE = "coverage.svg"
```

Comment:
remind people what this file is used for - eg 
Used to display test coverage in GH site in graphical form - is created upon test run